tests/integration/integration-test.sh

```diff
@@ -168,8 +175,27 @@ load_test() { #
   rm output.log
 }
 
+### CREATE IMAGEPULLSECRET ####################################
+create_imagepullsecret() { # NAMESPACE # CREATE
+  local CREATE=${2:-true}
+  if $CREATE; then
+    echo -n "Creating Namespace '${1}'..."
+    kubectl create ns ${1}
+  fi
+  if ! [[ -z "${imagepullsecret+x}" ]]; then
```

Comment:
Wouldn't 

```suggestion
  if [[ -n "${imagepullsecret+x}" ]]; then
```

also work and be way easier to comprehend?

Reply:
done